bridges/primitives/chain-asset-hub-rococo/src/lib.rs

```diff
@@ -46,6 +46,10 @@ frame_support::parameter_types! {
 	/// Base delivery fee to `BridgeHubRococo`.
 	/// (initially was calculated by test `BridgeHubRococo::can_calculate_weight_for_paid_export_message_with_reserve_transfer`)
 	pub const BridgeHubRococoBaseFeeInRocs: u128 = 1214739988;
+
+	/// User fee for ERC20 token transfer back to Ethereum.
+	/// (initially was calculated by test `OutboundQueue::calculate_fees` - ETH/ROC 1/411 and fee_per_gas 15 GWEI = 18679250000 + *25%)
+	pub const TransferERC20TokenBaseFeeInRocs: u128 = 23349062500;
```

Comment:
Not sure this is the best naming, I'm open to suggestions.

Reply:
I would suggest `BridgeHubEthereumBaseFeeInRocs` to be consistent with the Rocococ<->Wococo bridge.